.github/test-events/update-seed/push.json

```diff
@@ -0,0 +1,11 @@
+{
+  "push": {
+    "paths": [
+      "packages/seed/src/cli.ts",
+      "generators/php/sdk/src/cli.ts"
```

Comment:
Should the other languages be included here too? Is this in a state that you were using for testing?

Reply:
This is just for my own testing purposes; other languages could be tested like this, but not necessary. The thing that I'm testing is that it is triggered.